sites/partners/src/components/settings/PreferenceDrawer.tsx

```diff
@@ -524,22 +537,203 @@ const PreferenceDrawer = ({
               </FieldValue>
             </GridCell>
           </GridSection>
-          <GridSection columns={3} className={"mt-8"}>
+          <GridSection
+            title={t("settings.preferenceAdditionalFields")}
+            columns={4}
+            wrapperClassName={"border-t pt-8 mt-8"}
+          >
+            <GridCell span={1}>
+              <FieldValue label={t("settings.preferenceCollectAddress")}>
+                <>
+                  <FieldGroup
+                    name="collectAddress"
+                    type="radio"
+                    register={register}
+                    validation={{ required: true }}
+                    error={errors.collectAddress}
+                    fields={[
+                      {
+                        label: t("t.yes"),
+                        value: YesNoAnswer.Yes,
+                        defaultChecked: optionData?.collectAddress,
+                        id: "collectAddressYes",
+                        dataTestId: "collect-address-yes",
+                        inputProps: {
+                          onChange: () => {
+                            clearErrors("collectAddress")
+                          },
+                        },
+                      },
+                      {
+                        label: t("t.no"),
+                        value: YesNoAnswer.No,
+                        defaultChecked:
+                          optionData?.collectAddress !== undefined &&
+                          optionData?.collectAddress === false,
+                        id: "collectAddressNo",
+                        dataTestId: "collect-address-no",
+                        inputProps: {
+                          onChange: () => {
+                            clearErrors("collectAddress")
+                          },
+                        },
+                      },
+                    ]}
+                    fieldClassName="m-0"
+                    fieldGroupClassName="flex column items-center"
+                    dataTestId={"preference-option-collect-address"}
+                  />
+                </>
+              </FieldValue>
+            </GridCell>
+            <GridCell span={1}>
+              {collectAddressExpand && (
+                <FieldValue label={t("settings.preferenceValidatingAddress")}>
```

Comment:
The layout of this field is more compact than what the design has.
This code:
![image](https://github.com/bloom-housing/bloom/assets/42942267/7b12ee66-3b00-47cc-ba34-0b8686858ed2)
Design:
![image](https://github.com/bloom-housing/bloom/assets/42942267/857f0cd2-72f4-4cee-b413-c8ac5534c9d3)

Reply:
Thats true, i changed it to 3 columns, as it will match width of other fields. Now it looks like this 
<img width="1035" alt="image" src="https://github.com/bloom-housing/bloom/assets/9580543/4cc0fc77-5017-4733-99a2-b5c7b2cf3020">
FIgma container is slightly bigger than our in app, so it won't fit with 4 columns (hopefully there won't be another field on the right 😄 ).
Also @eajensenwa let me know what do you think about this solution?

Reply:
@slowbot pinging you for the design feedback here!

Reply:
This new change looks good. I did notice that the drawer gets wider when you select "yes" on the "Do you want to collect address information". I'm assuming that is because it has to make room fro the third column. I don't have an issue with that, just flagging it here